docs/csharp/language-reference/keywords/sizeof.md

```diff
@@ -9,13 +9,13 @@ helpviewer_keywords:
 ms.assetid: c548592c-677c-4f40-a4ce-e613f7529141
 ---
 # sizeof (C# Reference)
-Used to obtain the size in bytes for an unmanaged type. Unmanaged types include the built-in types that are listed in the table that follows, and also the following:  
+Used to obtain the size in bytes for an unmanaged type. Unmanaged types include the simple types that are listed in the table that follows, and also the following:  
```

Comment:
This is confusing, partly because the table follows much later and partly because the C# definition of unmanaged types is probably unfamiliar to most developers. It would be best use a list here:

> Unmanaged types include:
> - The simple types that are listed in the following table:
>   [table goes here]  
> - Enum types.   
> - Pointer types.
> - User-defined structs that do not contain any instance fields or auto-implemented instance properties that are reference types or constructed types.

(I think "auto-implemented instance properties" reads better than "instance auto-implemented properties".)